cmd/server/main.go

```diff
@@ -69,6 +70,14 @@ func realMain(ctx context.Context) error {
 		return fmt.Errorf("failed to process config: %w", err)
 	}
 
+	// Setup sessions
+	sessions := sessions.NewCookieStore(config.CookieKeys.AsBytes()...)
+	sessions.Options.Path = "/"
+	sessions.Options.Domain = "" // TODO: make configurable
```

Comment:
will be interesting to configure in a cloud run + custom domains scenario.

Reply:
It has to match what the user hits in the browser.

Reply:
Made configurable